src/runtime/json_parser.c

```diff
@@ -141,11 +141,11 @@ const char *flatcc_json_parser_space_ext(flatcc_json_parser_t *ctx, const char *
         ++buf;
     }
     while (buf != end && *buf <= 0x20) {
-        /* Fall through comments needed to silence gcc 7 warnings. */
         switch (*buf) {
         case 0x0d: buf += (end - buf > 1 && buf[1] == 0x0a);
             /* Consume following LF or treating CR as LF. */
-            fallthrough;
+            goto fallthrough;
+fallthrough:
         case 0x0a: ++ctx->line; ctx->line_start = ++buf; continue;
         case 0x09: ++buf; continue;
```

Comment:
Here case 0x09 need to be multiline and a label inserted between 0x0a: and ++ctx. Or perhaps better just copy the logic and avoid that mess.

Reply:
Copied the logic.